keadm/cmd/keadm/app/cmd/util/edgecoreinstaller.go

```diff
@@ -122,6 +123,15 @@ func (ku *KubeEdgeInstTool) createEdgeConfigFiles() error {
 	}
 	edgeCoreConfig.Modules.EdgeStream.TunnelServer = net.JoinHostPort(cloudCoreIP, strconv.Itoa(constants.DefaultTunnelPort))
 
+	if len(ku.Labels) >= 1 {
+		labelsMap := make(map[string]string)
+		for _, label := range ku.Labels {
+			key := strings.Split(label, "=")[0]
+			value := strings.Split(label, "=")[1]
```

Comment:
In the flag comment it's `:` to separate the key and value, here is `=`, which one does the user should chose?

Reply:
yeap I have fixed description and use '=' to replace ':'.I'm sorry for that :)

Reply:
lgtm, please fix the DCO issue :)

Reply:
ok,DCO had been fixed